api_v2/src/overtime/overtime.repository.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, ForbiddenException, NotFoundException } from '@nestjs/common';
+import { PrismaService } from '@/prisma/prisma.service';
+import { ApproveOvertimeRequestInput } from '@/graphql/graphql';
+import { PositionEnum } from '@/enums/position.enum';
+import { PositionHelper } from '@/enums/position.enum'; 
```

Comment:
you can combine them  in one

```suggestion
import { PositionEnum, PositionHelper } from '@/enums/position.enum';

```